<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarWindow</class>
 <widget class="QDialog" name="CalendarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="FrameCalendar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>284</width>
     <height>176</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCalendarWidget" name="CalendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>264</width>
      <height>156</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnSelectDate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
